llvm/lib/Target/WebAssembly/AsmParser/WebAssemblyAsmTypeCheck.cpp

```diff
@@ -112,9 +112,9 @@ bool WebAssemblyAsmTypeCheck::popRefType(SMLoc ErrorLoc) {
   return false;
 }
 
-bool WebAssemblyAsmTypeCheck::getLocal(SMLoc ErrorLoc, const MCInst &Inst,
+bool WebAssemblyAsmTypeCheck::getLocal(SMLoc ErrorLoc, const MCOperand &Op,
```

Comment:
Would it make sense to give these parameters a more-descriptive name, e.g. `SymbolOp` or something?

Reply:
Changed `Op` to `SymOp`(in `getSymRef`), `LocalOp` (in `getLocal`), `GlobalOp` (in `getGlobal`), and `SigOp` (in `getSignature`).